tests/Cache/Base.php

```diff
@@ -119,4 +119,13 @@ public function testFlush()
         $this->assertEquals(false, self::$cache->load('x', 100));
         $this->assertEquals(false, self::$cache->load('y', 100));
     }
+
+    /**
+     * General tests
+     * Can be overwritten in specific adapter if required, such as None Cache
```

Comment:
What does this mean? I dont think we override it in None cache. Got me confused a little.

Reply:
Removed comments, probably copy-pasted.